nrcan_etl/src/energuide/embedded/evaluation_type.py

```diff
@@ -14,6 +14,6 @@ def from_code(cls, code: str) -> 'EvaluationType':
             return EvaluationType.PRE_RETROFIT
         elif code == cls.POST_RETROFIT.value:
             return EvaluationType.POST_RETROFIT
-        elif code == cls.INCENTIVE_PROGRAM:
+        elif code == cls.INCENTIVE_PROGRAM.value:
```

Comment:
whoa, shouldn't there have been a test that caught this?

Reply:
This is the test that caught it.
I agree it should have existed before, but it was missed.